snc-library.sh

```diff
@@ -283,6 +283,7 @@ function wait_till_cluster_stable() {
        fi
        if [ $count -eq $numConsecutive ]; then
            echo "Cluster has stabilized"
+           retry ${OC} delete pods --field-selector=status.phase=Failed -A
```

Comment:
Not 100% clear from the commit log what happens exactly. It's `retry all_pods_are_running_completed "${ignoreNamespace}"` which would fail eventually? Maybe this function should just ignore failed pods?

Reply:
> It's `retry all_pods_are_running_completed "${ignoreNamespace}"` which would fail eventually? 

No, this is just ignore all the pods from that namespace irrespective of the state of pod (passed/failed/Error)

> Maybe this function should just ignore failed pods?

No, We can't just ignore any failed pods (like ignoring api-server or any core-component pods result to unstable cluster) This is just removing the failed pods which are stuck in failed state (mostly of those are some component complete pods or garbage collector one).